fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -763,6 +766,26 @@ private String mintNewPid(final String slug) {
         return pid;
     }
 
+    /**
+     * perform save with checking to prevent SN-Sibling nodes from creating
+     * @param resource the FedoraResource need to save
+     * @throws javax.jcr.AccessDeniedException
+     * @throws VersionException
+     * @throws LockException
+     * @throws ConstraintViolationException
+     * @throws RepositoryException
+     */
+    private synchronized void save(final FedoraResource resource) throws javax.jcr.AccessDeniedException,
```

Comment:
I am not sure making this a static method would be an improvement. My understanding is that without this `synchronized` block or method, the issue is not resolved. Is that true?
For example, do your integration tests fail if you remove the keyword `synchronized` here?